blog.md

```diff
@@ -8,69 +8,56 @@ excerpt: ""
 Author: Kallinteris Andreas
 thumbnail: 
 image: 
-read_time: ???
+read_time: 4 minutes
 ---
 
-
 # Introduction
-`Gymnasium/MuJoCo` is a set of reinforcement learning environments where the goal is to make a robot locomote as fast as possible.
+`Gymnasium/MuJoCo` is a set of robotics based reinforcement learning environments using the [mujoco](https://mujoco.org/) physics engine with various different goals for the robot to learn: standup, run quickly, move an arm to a point.
 
-The version 0 of `Gymnasium/MuJoCo` was introduced in `gym==0.0.1` 2016,
-followed shortly after by version 1 (`gym==0.1.0` in 2016) which addressed several configuration errors.
-Version 2 (`gym==0.9.5` in 2018), which brought major backend improvements using the `mujoco-py=>1.5` simulator. 
-Version 3 (`gym==0.12.0` in 2019) offers increased customization options, enabling the user to modify parts of the environment, such as the reward function, and make slight adjustments to the robot model.
-Version 4 (`gym==0.24.0` in 2020) ported the environments to the new `mujoco>=2.2.0` simulator. However, it removed the capability of slightly modifying the environment.
+## A quick history of MuJoCo environments 
+Originally introduced as version 0 in `gym==0.0.1` way back in 2016, this was shortly followed by version 1 (`gym=0.1.0`) to address several configuration errors. 
+In 2018, version 2 in `gym=0.9.5` was released, which brought major backend improvements using `mujoco-py=>1.5` simulator. 
+Version 3 (`gym=0.12.0` in 2018) offered increased customization options, enabling users to modify parts of the environment such as the reward function and slight adjustments to the robot model. 
+With Google-DeepMind buying MuJoCo, open sourcing the code and releasing a dedicated python module (`mujoco`), version 4 ports the environments to the new `mujoco>=2.2.0` simulator however removed the capability to slightly modify the environment. 
 
-Numerous RL research papers have utilized Gymnasium/MuJoCo environments, including Hopper, Ant, and Humanoid. As a standard set of difficult-to-solve robotic environments, they have been a cornerstone in the development and evaluation of RL methods.
+## MuJoCo v5
```

Comment:
Rereading this, I feel this whole section should be moved into the introduction as it is a summary what we have do

Reply:
so you want the subsection "A quick history of MuJoCo environments" to be after "MuJoCo v5" subsection?

Reply:
Maybe not, I will have another look